### DISCUSSION TITLE
CP2K-MD data process

### DISCUSSION THREAD

**Discussion by LiuJDinCS** [deepmodeling/dpdata#137] [Q&A]
Dear developer：
If I use CP2K to run MD, traj, energy and force are put in different files (cp2k-pos-1.xyz, cp2k-1.ener, cp2k.out). How to use dpdata to process these data?

**Comment by amcadmus** [comment]
Have you checked the readme of dpdata? Does it help?

**Reply by rajnichahal** [reply]
Hello. dpdata readme doesn't discuss the conversion of cp2k MD trajectories from files commonly written in cp2k simulation (.xyz, .ener, and .forces). I think it only goes over the conversion procedure if somebody has kept both .log and pos.xyz files.

**Reply by robinzyb** [reply]
indeed, however the log file must be kept since it has cell informations in it

**Reply by lizhiqiang778** [reply]
> indeed, however the log file must be kept since it has cell informations in it

hi yongbin, I deal with  my cp2k trajs to using dpdata ,  and  I can get energy , coord and box but forces. Could you help to check it?

**Comment by lmqnuaa** [comment]
Hello, you just need to put the file ' pos.xyz' and ' .log' in the same folder, the name of the folder, for example, is "test", then you activate python, >>>import dpdata, then >>>dpdata.LabeledSystem("**/test", fmt='cp2k/aimd_OUTPUT'), dpdata will automically recognize the files in the folder.

**Reply by PolyuWeldingSpock** [reply]
Hello, I want to know how to convert .dcd format trajectory files from cp2k to DeepMD toolkit? Because I run a AIMD in NPT ensemble, so I need box size formation. As a result I have to output .dcd format but not .xyz format because .xyz does not have any information of box.

**Reply by robinzyb** [reply]
hello @PolyuWeldingSpock, I haven't implemented for .dcd. and am not familiar with it. Could you please provide me an input, .dcd with several snapshots and an output to my email yongbin.zhuang@epfl.ch

**Reply by PolyuWeldingSpock** [reply]
hello @robinzyb , I have sent it to your email. Please check. Thanks for your help.
